fitbenchmarking/plotHelper.py

```diff
@@ -64,48 +65,25 @@ def order_data(self):
 
         for j in range(0,len(yData)):
             for k in range(0,len(xData)-1):
-                if xData[k]>xData[k+1]:
-                    xData[k+1],xData[k]=xData[k],xData[k+1]
-                    yData[k+1],yData[k]=yData[k],yData[k+1]
-                    eData[k+1],eData[k]=eData[k],eData[k+1]
+                if xData[k] > xData[k+1]:
+                    xData[k+1], xData[k] = xData[k], xData[k+1]
+                    yData[k+1], yData[k] = yData[k], yData[k+1]
+                    eData[k+1], eData[k] = eData[k], eData[k+1]
 
 
-class insert(data):
-    """
-    Minimal information for defining an insert
-    """
-    def __init__(self):
-        self.logs= {'x':False,'y':False}
-        #make this a list later
-        self.data = []
-        self.labels={'x':"",'y':'',"title":""}
-        self.xrange={'start':0.0,'end':0.0}
-        self.yrange={'start':0.0,'end':0.0}
-        self.position={'left':0.0, 'bottom':0.0, 'width':0.25, 'height':0.25}
-
-    def add_data(self,inputData):
-        """
-        Creates a horizontal bar plot
-        @param safe :: if to ensure that y labels only appear once
-        @param save:: the name of the file to save to
-        the default is not to save
-        """
-        self.data.append(inputData)
-
-
-class plot(data,insert):
+class plot(data):
     """
     Minimal information for a plot
```

Comment:
Would you be able to expand on "Minimal information for a plot" documentation please